server/src/test/java/io/crate/types/ArrayTypeTest.java

```diff
@@ -48,8 +48,14 @@ public DataType<List<Object>> getType() {
     }
 
     @Override
-    protected boolean supportsDocValues() {
-        return false;
+    @SuppressWarnings("unchecked")
+    protected DataDef<List<Object>> getDataDef() {
+        // Exclude float vectors and objects
```

Comment:
why do we exclude objects?

Reply:
Because the test framework doesn't generated nested object data properly.  The ObjectTypeTest handles this explicitly so I don't think it's necessary in the array test?  I'll add a comment.

Reply:
ok, cool, didn't check deeper, thx!